pkgs/tools/wayland/sway-audio-idle-inhibit/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, libpulseaudio
+, wayland
+, wayland-protocols
+}:
+stdenv.mkDerivation rec {
+  pname = "sway-audio-idle-inhibit";
+  version = "0.1.1-unstable-2023-08-09";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwayAudioIdleInhibit";
+    rev = "c850bc4812216d03e05083c69aa05326a7fab9c7";
+    sha256 = "sha256-MKzyF5xY0uJ/UWewr8VFrK0y7ekvcWpMv/u9CHG14gs=";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkg-config
+    libpulseaudio wayland wayland-protocols
+  ];
```

```suggestion
  nativeBuildInputs = [
    meson ninja pkg-config
  ];
  
  buildInputs = [
    libpulseaudio wayland wayland-protocols
  ];
```
Binaries from these packages aren't used in the build

---

Thanks for the corrections! I fixed everything requested.